projects/fsastorefrontlib/src/cms-components/fnol/fnol-navigation/fnol-navigation.component.ts

```diff
@@ -109,15 +109,21 @@ export class FNOLNavigationComponent implements OnInit, OnDestroy {
       ])
         .pipe(
           map(([submittedFormData, uploadedContent]) => {
+            let newClaimData = { ...claimData };
```

Comment:
same here, its enough just to name it claim for example

Reply:
`claimCopy` or `claimClone` to create some uniformity for these kind of cases? At least temporary, I jope...